sailor/assetcentral/utils.py

```diff
@@ -127,15 +127,18 @@ def _fetch_data(endpoint_url, unbreakable_filters=(), breakable_filters=()):
     filters = _compose_queries(unbreakable_filters, breakable_filters)
     service = OAuthFlow('asset_central')
 
-    result = []
-    for filter_string in filters:
-        parameters = {'$filter': filter_string} if filter_string else None
-        endpoint_data = service.fetch_endpoint_data(endpoint_url, method='GET', parameters=parameters)
+    if not filters:
+        result = service.fetch_endpoint_data(endpoint_url, method='GET')
```

Comment:
if the return value is not a list the result is now different depending on whether we do or don't have filters (the `append` case below).
not sure how to do this nicely, because we don't want to repeat the `isinstance` test up here I think (but it's not worth a (local) function either? 
Of course we could also just introduce a
``` 
if not filters:
  filters = ['']
```
(or `filters = [None]`) here, that brings us back to the old behaviour and isn't great either...

Reply:
I'm not sure I get your concern. It's about the return type depending on the filters/ not filters?

Then maybe this solves the problem?:
```
result = []
if not filters:
    result.extend(service.fetch_endpoint_data(endpoint_url, method='GET'))
```